flowetl/tests/integration/test_fixuid.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Tests that fixuid works as expected
+"""
+
+
+def test_uid(docker_client, flowetl_tag):
+    """
+    test that we can run the flowetl container with a specific user
+    """
+
+    user = "1002:1003"
+    out = docker_client.containers.run(
+        f"flowminder/flowetl:{flowetl_tag}", "bash -c 'id -u'", user=user
+    )
+    assert out.decode("utf-8").strip() == "1002"
+
+
+def test_gid(docker_client, flowetl_tag):
+    """
+    test that we can run the flowetl container with a specific user
+    """
+
+    user = "1002:1003"
+    out = docker_client.containers.run(
+        f"flowminder/flowetl:{flowetl_tag}", "bash -c 'id -g'", user=user
+    )
+    assert out.decode("utf-8").strip() == "1003"
+
+
+def test_uid_is_airflow(docker_client, flowetl_tag):
```

Comment:
Does this test subsume the previous one? No harm having both, but if this one does the same thing as the other one (plus a bit more, because it explicitly tests for the username too) then we can remove the previous one (although on second thought I'm not sure this one actually _tests_ the gid, so perhaps best to keep both).

Reply:
Well `--user X:Y` will always result in the first two tests passing regardless of image. But the third is taking the UID of the running user and making sure that the user is called 'airflow'. So perhaps only need UID test and this one. Make sure that the UID is what we expect and then check that this UID is the `airflow` user?